Feature/admin api charts (#972)
neicnordic/sensitive-data-archive: charts/sda-svc/README.md

`global.backupArchive.volumePath` | Path to the mounted `posix` volume. |`/backup`
`global.backupArchive.nfsServer` | URL or IP address to a NFS server. |`""`
`global.backupArchive.nfsPath` | Path on the NFS server for the backup archive. |`""`
`global.api.adminFileSecret` | A secret holding a JSON file named `admin.json` containg a list of identifiers |``

> `global.api.adminUsers` | A list of user identifiers |``

Reviewer: ```suggestion
`global.api.adminUsers` | A list of identifiers of the users with admin privileges |``
```

> `global.api.jwtPubKeyName` | Public key ues to verify the JWT. |``

Reviewer: ```suggestion
`global.api.jwtPubKeyName` | Public key used to verify the JWT. |``
```

Parameter | Description | Default
--------- | ----------- | -------

> `api.replicaCount` | desired number of replicas | `2`

Reviewer: ```suggestion
`api.replicaCount` | Desired number of replicas | `2`
```
